jwst/resample/resample.py

```diff
@@ -71,6 +71,9 @@ def __init__(self, input_models, output=None, ref_filename=None, **pars):
         self.output_wcs = resample_utils.make_output_wcs(self.input_models)
         log.debug('Output mosaic size: {}'.format(self.output_wcs.data_size))
         self.blank_output = datamodels.DrizProductModel(self.output_wcs.data_size)
+
+        # update meta data and wcs
+        self.blank_output.update(input_models[1])
```

Comment:
Just curious, why index 1 in input_models, rather than 0?

Reply:
Doh!  *brainfart*  That's a bug.  I should be tagging you to review all my PRs.

Reply:
You actually forgot that python is zero-indexed?! Shame, shame, shame ...

Reply:
I know.  I should probably stop working right now.